engine/chaos/cases/case_fake_job.go

```diff
@@ -20,9 +20,9 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/pingcap/log"
```

Comment:
Why move this line from the third group here?

Reply:
For tiflow, this line is the third-party package.